web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1680,6 +1680,10 @@
         return gnConfig["system.feedback.mailServer.hostIsDefined"];
       };
 
+      $scope.getBatchEditingAccessLevel = function() {
```

Comment:
Why not using this approach adding a method to the user information: 

https://github.com/geonetwork/core-geonetwork/blob/67b633f137905613a718e2c675a93ce39679944c/web-ui/src/main/resources/catalog/js/CatController.js#L1830-L1837

That should simplify the changes in the html code with something similar to:

```
data-ng-if="user.canBatchEditMetadata()"
```

Reply:
yes, this is better. I have made the code change for this